dashboard/src/modules/components/TableComponent/index.jsx

```diff
@@ -31,8 +30,10 @@ import NavItems from "../NavbarComponent";
 import EmptyTable from "../EmptyStateComponent";
 import moment from "moment";
 import { fetchPublicDatasets } from "../../../actions/fetchPublicDatasets";
-let startDate = moment(new Date(1990, 10, 4)).format("YYYY/MM/DD");
-let endDate = moment(new Date(2040, 10, 4)).format("YYYY/MM/DD");
+import TablePagination from "../PaginationComponent";
+import { formatDate } from "../../../utils/dateFormatter";
+let startDate = formatDate(new Date(1990,10,4));
+let endDate = formatDate(new Date());
```

Comment:
This is the same setup which `DatePicker` is doing.  Shouldn't they use common code?